Verifiable Credentials JSON Schema Specification.md

```diff
@@ -0,0 +1,469 @@
+
+#### JSON Schema
+<details>
+  <summary>Show/Hide JSON Schema</summary>
+
+```
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "type": {
+      "type": "string"
+    },
+    "modelVersion": {
+      "type": "string"
+    },
+    "name": {
+      "type": "string"
+    },
+    "author": {
+      "type": "string"
+    },
+    "authored": {
+      "type": "string"
+    },
+    "schema": {
+      "type": "object",
+      "properties": {
+        "$schema": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string"
+        },
+        "properties": {
+          "type": "object"
+        },
+        "required": {
+          "type": "array",
+          "items": [
+            {
+              "type": "string"
+            }
+          ]
+        },
+        "additionalProperties": {
+          "type": "boolean"
+        }
+      },
+      "required": [
+        "$schema",
+        "description",
+        "type",
+        "properties",
+        "required",
+        "additionalProperties"
+      ]
+    },
+    "proof": {
+      "type": "object",
+      "properties": {
+        "created": {
+          "type": "string"
+        },
+        "creator": {
+          "type": "string"
+        },
+        "nonce": {
+          "type": "string"
+        },
+        "signatureValue": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "created",
+        "creator",
+        "nonce",
+        "signatureValue",
+        "type"
+      ]
+    }
+  },
+  "required": [
+    "type",
+    "modelVersion",
+    "name",
+    "author",
+    "authored",
+    "schema",
+    "proof"
+  ]
+}
+```
+
+</details>
+
+#### Example
+```json
+{
+  "type": "https://credentials.workday.com/docs/credential-schema.json",
+  "modelVersion": "1.0",
+  "id": "did:work:MDP8AsFhHzhwUvGNuYkX7T;id=06e126d1-fa44-4882-a243-1e326fbe21db;version=1.0",
+  "name": "EmailCredentialSchema",
+  "author": "did:work:MDP8AsFhHzhwUvGNuYkX7T",
+  "authored": "2018-01-01T00:00:00+00:00",
+  "schema": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "description": "Email",
+    "type": "object",
+    "properties": {
+      "emailAddress": {
+        "type": "string",
+        "format": "email"
+      }
+    },
+    "required": ["emailAddress"],
+    "additionalProperties": false
+  },
+  "proof": {
+      "created": "2019-09-27T06:26:11Z",
+      "creator": "did:work:MDP8AsFhHzhwUvGNuYkX7T#key-1",
+      "nonce": "0efba23d-2987-4441-998e-23a9d9af79f0",
+      "signatureValue": "2A7ZF9f9TWMdtgn57Y6dP6RQGs52xg2QdjUESZUuf4J9BUnwwWFNL8vFshQAEQF6ZFBXjYLYNU4hzXNKc3R6y6re",
+      "type": "Ed25519VerificationKey2018"
```

Comment:
This signature still looks very close to JSON-LD Signature. I'd like to recommend we either use JSON-LD signatures with detached jws (such as this example): https://github.com/decentralized-identity/lds-ecdsa-secp256k1-2019.js#verify

Or we find a way to use another form of detached jws without JSON-LD, pre processing. But I think thats better to tackle in a subsequent PR.

Reply:
Agree, and agree this should be in a subsequent PR

Reply:
I have captured this discussion in the following issue: https://github.com/w3c-ccg/vc-json-schemas/issues/2

I'm marking this as resolved, and we should be ready to merge once we hear back on PR comment duration.